packages/wesley-generator-supabase/PROGRESS.md

```
@@ -0,0 +1,19 @@
+# @wesley/generator-supabase — Progress
+
+## MVP (idea works)
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)